File: Examen_prueba3/app/src/main/java/com/example/examen_prueba3/Facultad.java
package com.example.examen_prueba3;

import com.google.firebase.database.IgnoreExtraProperties;

@IgnoreExtraProperties
public class Facultad {
    private String nombre;
    private String creditos;
    private String metodologia;

    public Facultad() {
    }

    public Facultad(String nombre, String creditos, String metodologia) {
        this.nombre = nombre;
        this.creditos = creditos;
        this.metodologia = metodologia;
    }

    public String getNombre() {
        return nombre;
    }

    public String getCreditos() {
        return creditos;
    }

    public String getMetodologia() {
        return metodologia;
    }
}

File: PRIMER PARCIAL/Examen_prueba1/app/src/main/java/com/example/facultad_1p/Facultad.kt
package com.example.facultad_1p
import com.google.firebase.firestore.Exclude
import com.google.firebase.firestore.IgnoreExtraProperties

@IgnoreExtraProperties
data class Facultad(val nombre: String? = null, val creditos: String? = null, val metodologia: String? = null, val url: String? = null, @Exclude val key: String? = null) {
}